pkg/nfs/nodeserver.go

```diff
@@ -36,7 +37,19 @@ type nodeServer struct {
 }
 
 func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolumeRequest) (*csi.NodePublishVolumeResponse, error) {
+	klog.V(2).Infof("NodePublishVolume called with request %v", *req)
```

Comment:
Should we use something like https://github.com/kubernetes-csi/csi-lib-utils/blob/master/connection/connection.go#L181 to log all csi calls? The other advantage is that it uses protosanitizer which strips secrets from being logged.